importer/api/__init__.py

```diff
@@ -124,6 +125,7 @@ def import_course(bundle, repo_id, user_id, static_dir):
     )
     import_static_assets(course, static_dir)
     import_children(course, src, None)
+    populate_xanalytics_fields.delay(course.id)
```

Comment:
Isns't this already inside an asynchronous task, why are we asyncing on top of asyncing, ouch my brain ;)